libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/thrift/ThriftValidate.java

```diff
@@ -269,6 +270,9 @@ private static void validateLicenseIdMatch(License license) throws SW360Exceptio
     public static void prepareProject(Project project) throws SW360Exception {
         assertNotEmpty(project.getName());
         project.setType(TYPE_PROJECT);
+        if(!project.isSetClearingState()) {
```

Comment:
We usually have a space after `if`:
```suggestion
        if (!project.isSetClearingState()) {
```

Reply:
Done.